cmd/security-secretstore-setup/entrypoint.sh

```diff
@@ -44,6 +44,12 @@ fi
 echo "$(date) Changing ownership of ${EDGEX_SECRETS_ROOT} to ${EDGEX_USER}:${EDGEX_GROUP}"
 chown -Rh ${EDGEX_USER}:${EDGEX_GROUP} "${EDGEX_SECRETS_ROOT}"
 
+ekuiper_uid="${EDGEX_EKUIPER_USER_ID:-1001}"
+ekuiper_gid="${EDGEX_EKUIPER_GROUP_ID:-1001}"
+
+echo "$(date) Changing ownership of ${EDGEX_SECRETS_ROOT}/rules-engine to ${ekuiper_uid}:${ekuiper_gid} to accommodate ekuiper."
+chown -Rh ${ekuiper_uid}:${ekuiper_gid} "${EDGEX_SECRETS_ROOT}/rules-engine"
```

Comment:
please follow our coding convention as other parts.  use upper case for these variables

Reply:
i left them lowercase in order to demonstrate they are local and not env vars, but i can capitalize them, sure.

Reply:
I agree with your style. We may use this naming convention for the future new script, and apply to the existing script if possible